docs/deployment/backstage-deploy/aws.md

```diff
@@ -0,0 +1,188 @@
+        "lightsail:GetKeyPairs",
+        "lightsail:CreateContainerServiceDeployment",
+        "lightsail:GetContainerServiceMetricData",
+        "lightsail:GetContainerAPIMetadata",
+        "lightsail:DeleteContainerService",
+        "lightsail:GetContainerLog",
+        "lightsail:TagResource"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "Statement1",
+      "Effect": "Allow",
+      "Action": [],
+      "Resource": []
+    }
+  ]
+}
+```
+
+Then click on `Next` and give the policy a name and a description of your liking. Afterwards, click on `Create policy`.
+
+- Navigate back to the user creation window and press on the refresh button and search for the policy you just created. Now, create the user.
+- Now you will be redirected to all users, click on the user you just created and click on `Security credentials`
+- Scroll below and click on `Create access key`
+- Choose `Command Line Interface (CLI)`
+- Now export the following values
+
+```bash
+$ export AWS_ACCESS_KEY_ID=... (first value)
+$ export AWS_SECRET_ACCESS_KEY=.... (second secret value)
+```
+
+## Configuring the Pulumi CLI
+
+Second, install the [Pulumi CLI](https://www.pulumi.com/docs/get-started/install/).
+
+Then we need to execute the following commands, to set Pulumi up:
+
+```bash
+$ pulumi login --local
+$ export PULUMI_CONFIG_PASSPHRASE="<your-secret>"
+```
+
+By using `pulumi login --local` we are making sure that Pulumi stores our state on the local file disk. The environment variable `PULUMI_CONFIG_PASSPHRASE` is used by Pulumi to generate a unique key for your stack
+
+## Deploying your instance on Lightsail
+
+:::warning
+
+Make sure that [Docker](https://docs.docker.com/) is running before you start with this section.
+
+:::
+
+After you have made your changes to your local instance, it's time to deploy it on Lightsail.
+
+First, we need to configure the `app-config.yaml` and update the `baseUrl`.
+
+```diff
+app:
+-  baseUrl: http://localhost:3000
++  baseUrl: ${BACKSTAGE_HOST}
+
+backend:
+-  baseUrl: http://localhost:7007
++  baseUrl: ${BACKSTAGE_HOST}
+  listen:
+    port: 7007
+```
+
+The environment variable `BACKSTAGE_HOST` will be set to the endpoint that AWS Lightsail creates.
+
+In addition, you should create a `app-config.local.yaml`:
+
+```bash
+$ touch app-config.local.yaml
+```
+
+And then update the file with the following yaml:
+
+```yaml
+app:
+  baseUrl: http://localhost:3000
+
+backend:
+  baseUrl: http://localhost:7007
+```
+
+Now we can deploy our instance!
+
+```bash
+$ npx backstage-deploy aws --stack backstage-poc --create-dockerfile
+```
+
+In the first part of the command, we are specifying that we want to deploy our instance on AWS. With the [`--stack`](https://www.pulumi.com/docs/reference/cli/pulumi_stack/) option, we are providing Pulumi a reference to our container deployment. Furthermore, with the `--create-dockerfile` option, there will be a `Dockerfile` and `.dockerignore` created in the root of the project.
```

Comment:
Curious about `--create-dockerfile` here, we already ship a `Dockerfile` with `@backstage/create-app`, how come we go this route?

Reply:
Yeah it's optional for if an user doesn't have a `Dockerfile` provided. I just wanted to display a full command  in action, in reality you could just do `npx backstage-deploy aws` or `npx backstage-deploy --dockerfile <path>` .